src/lib389/lib389/cli_ctl/dblib.py

```diff
@@ -236,7 +236,7 @@ def dblib_bdb2mdb(inst, log, args):
         total_dbi += be['dbi']
 
     # Round up dbmap size
-    dbmap_size = DEFAULT_DBMAP_SIZE
+    dbmap_size = DEFAULT_LMDB_SIZE * GIGABYTE
     while (total_dbsize * DBSIZE_MARGIN > dbmap_size):
         dbmap_size *= 1.25
```

Comment:
I'm not sure what this is trying to achieve? maybe some better comments about this algo and why.

Reply:
When looking at the whole function, it is clear that this code computes a size for lmdb database that is large enough to store the existing data.
IMHO the puzzling question is not what it tries to achieve but why it is done this way 
   (starting from the default size and increasing the size by steps until it is large enough) 
    rather than using directly the computed size plus a margin ?
 There was probably a reason but I just do not remember it and whatever it was it is likely arguable
 But that piece of code was written 2 years ago and the only thing that changed is the way to get the default value. So my feeling is that we are getting out of the scope of this change and if you think we should really change things in this area, we should better open a new issue.

Reply:
Anyway I have to review that code to fix a regression due to the change of default db map size from 2Gb to 20Gb 
So I spend the time to understand what I have done 2 years ago ! -;)  
Then I added a few comments, especially on the tests done after the size computation.